### PR TITLE
Redirect user to the "root" page

### PR DESCRIPTION
**Pull request by BunducPaul**

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -12,7 +12,9 @@ function NavBar() {
   return <AppBar position="static">
     <Toolbar>
       <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
-        SalesWell
+        <Link href="/" passHref>
+          <a className='navHome'> SalesWell</a>
```
Comment (S7012MY) [inline components/Navbar.tsx]: you have an extra space here

**Review by S7012MY — CHANGES_REQUESTED** [review]

File: components/Navbar.tsx [hunk 2]
```diff
@@ -11,10 +11,14 @@ function NavBar() {
 
   return <AppBar position="static">
     <Toolbar>
-      <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
-        SalesWell
+      <Typography component="div" sx={{ flexGrow: 1 }}>
```
Comment (keepcoding-tech) [inline components/Navbar.tsx]: Can you get rid of this element? You don't really need two `Typography` elements here.

File: components/Navbar.tsx [hunk 3]
```diff
@@ -11,10 +11,14 @@ function NavBar() {
...
+        <Link href="/" passHref>
+          <Typography variant="h6" component='a' color="inherit" sx={{ textDecoration: 'none' }}>
```
Comment (keepcoding-tech) [inline components/Navbar.tsx]: Let's add the `color` attribute inside the `sx` (style) attribute.

File: components/Navbar.tsx [hunk 4]
```diff
@@ -11,10 +11,14 @@ function NavBar() {
...
+            SalesWell
+          </Typography>
+        </Link>
       </Typography>
-      { session ?
```
Comment (keepcoding-tech) [inline components/Navbar.tsx:17]: Why did you remove these spaces?

File: components/Navbar.tsx [hunk 5]
```diff
@@ -26,13 +30,13 @@ function NavBar() {
...
             </Button>
           </Link>
-          <Button color="inherit" id = "logout-button" onClick={() => signOut()}>
+          <Button color="inherit" id="logout-button" onClick={() => signOut()}>
             Logout
           </Button>
         </>
         : <Button color="inherit" id="login-button" onClick={() => signIn()}>
           Login
-        </Button> }
```
Comment (keepcoding-tech) [inline components/Navbar.tsx:35]: Same here?!

**Review by keepcoding-tech — CHANGES_REQUESTED** [review]

File: components/Navbar.tsx (hunk 2 above)
Comment (BunducPaul) [reply components/Navbar.tsx]: If I remove it, there will be clickable empty space between the link and the buttons, which is not what I want.

File: package.json [hunk 6]
```diff
@@ -53,7 +53,7 @@
     "libphonenumbers": "^1.3.0",
     "mock-aws-s3": "^4.0.2",
     "moment": "^2.29.3",
-    "next": "^12.0.10",
+    "next": "^12.3.4",
```
Comment (keepcoding-tech) [inline package.json]: Please don't modify this file, a different version can affect other dependencies, so we don't want to take that risk.

File: components/Navbar.tsx (hunk 2 above)
Comment (keepcoding-tech) [reply components/Navbar.tsx]: Ok, so how can we keep the same interface but not make the space between the text and buttons clickable?

**Review by keepcoding-tech — CHANGES_REQUESTED** [review]

File: components/Navbar.tsx [hunk 7]
```diff
@@ -10,29 +10,33 @@ function NavBar() {
...
-        SalesWell
-      </Typography>
-      { session ? 
-        <>
-          <Link href="/booking/list">
-            <Button color="inherit">
-              Bookings
+    <Toolbar sx={{ display: "flex", justifyContent: "space-between" }}>
+      <Link href="/" passHref>
+        <Typography variant="h6" component='a' sx={{ textDecoration: 'none', color: "inherit" }}>
```
Comment (keepcoding-tech) [inline components/Navbar.tsx]: Write the `sx` attribute on the next line. Make sure that the line does not go more than 80 characters.
Also, the `sx` should be indented with one tab to the right if written on the next line.

**Review by keepcoding-tech — CHANGES_REQUESTED** [review]

**Review by keepcoding-tech — APPROVED** [review]

**Review by S7012MY — APPROVED** [review]

**Pull request merged**